docs/protocol-design/work.md

```diff
@@ -14,16 +14,51 @@ Existing content:
 
 ---
 
-## Spam resistance focus
+## Spam resistance
+
+!!! quote ""
+	Proof-of-Work (PoW) is used as a method to deter spam transactions on the network. It is **not** used in the consensus mechanism.
+
+A spam transaction is a block broadcasted with the intention of saturating the network, reducing its available throughput for other network participants, or increasing the size of the ledger. Participants can compute the required PoW in the order of seconds. The cost of spamming the network increases linearly with the number of spam transactions, thus reducing the impact of spam transactions from theoretically infinite to a manageable amount.
 
```

Comment:
Although it is further down, perhaps worth mentioning here that this is only effective when work validation is sufficiently trivial to perform, otherwise resources used to verify the work could impact performance as much as a larger volume of spam transactions that didn't require work.